lib/index.js

```diff
@@ -356,16 +368,95 @@ function Setup(T, { checkTypes , _env=[] }){
 
     const Class =
         def(
-            'UnionType.Class'
+            'SumType.Class'
             ,{}
             ,[T.String, T.StrMap(T.Any), T.Object, T.Any]
-            ,CreateUnionType
+            ,CreateSumType
         )
 
+    const $SumType = T.UnaryType(
+        'JAForbes/SumType'
+        ,''
+        , x => [x]

```

Comment:
I find your code style intriguing. Under what circumstances do you include a space after a leading comma?